.github/tracing/tempo.yaml

```diff
@@ -0,0 +1,24 @@
+server:
+  http_listen_port: 3200
+
+distributor:
+  receivers:                           
+    otlp:
+      protocols:
+        http:
+        grpc:
+
+ingester:
+  max_block_duration: 5m               # cut the headblock when this much time passes. this is being set for demo purposes and should probably be left alone normally
+
+compactor:
+  compaction:
+    block_retention: 1h                # overall Tempo trace retention. set for demo purposes
+
+storage:
+  trace:
+    backend: local                     # backend configuration to use
+    wal:
+      path: /tmp/tempo/wal             # where to store the the wal locally
```

Comment:
any concerns about the violate nature of /tmp or disk limits?

Reply:
This file will only be used when spinning up a fresh docker container for tempo, and exists only for one PR, so I'm not worried about disk limits and other resource constraints. If we hit resource constraints we can update how we spin up the docker container